website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,12 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## September 2024
 
```

Comment:
non blocker but i wonder if we can make it explicit that these are changes for versionless users and upcoming 1.9? maybe its best to keep it simple but wanted to flag that!

e.g 

#### dbt Cloud Versionless (coming in 1.9 release)
- your bulk updates
#### Also available this month
rest of the updates for sept